### PR TITLE
[ui][fix] fixing api connector tests

### PR DESCRIPTION
**Pull request by mcada**
//test: @api-connector,@api-connector-invalid-swagger

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java [hunk 1]
```diff
@@ -69,7 +73,11 @@ public void deleteConnector(String name) {
 
     @Then("^uploads? swagger file (.+)$")
     public void uploadSwaggerFile(String filePath) {
-       uploadSwagger(new ArrayList<List<String>>(Arrays.asList(Arrays.asList("file", filePath))));
+        //uploadSwagger(new ArrayList<List<String>>(Arrays.asList(Arrays.asList("file", filePath))));
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java]: delete if not used

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java [hunk 2]
```diff
@@ -69,7 +73,11 @@ public void deleteConnector(String name) {
...
+        log.info("File path: " + filePath);
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java]: debug?

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java [hunk 3]
```diff
@@ -69,7 +73,11 @@ public void deleteConnector(String name) {
...
+        ElementsCollection col = $$(By.tagName("input")).filter(attribute("type", "file"));
+        Assertions.assertThat(col.size()).isEqualTo(1);
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java]: static import

Comment (avano) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/connectors/ApiClientConnectorsSteps.java]: assertThat(col).size().isEqualTo(1)

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**